.env.example

```
@@ -0,0 +1,9 @@
+MONOREPO_DIR=../optimism
+SUPERCHAIN_TARGET=mainnet
+CHAIN_NAME=awesomechain
+DEPLOYMENTS_DIR=${MONOREPO_DIR}/packages/contracts-bedrock/deployments/getting-started
+ROLLUP_CONFIG=${MONOREPO_DIR}/op-node/rollup.json
+GENESIS_CONFIG=${MONOREPO_DIR}/op-node/genesis.json
+PUBLIC_RPC="http://awe.some.rpc"
+SEQUENCER_RPC="http://awe.some.seq.rpc"
+EXPLORER="https://awesomescan.org"
```

What do you think of either renaming vars (first suggestion below), or adding comments (second suggestion below) to clarify when we need L1 vs. L2 data? Renaming vars is likely a breaking change for existing PRs like #107, so I'm ok just adding comments for now and tracking renames in #111

```suggestion
MONOREPO_DIR=../optimism
L1_CHAIN_NAME=mainnet
L2_CHAIN_NAME=awesomechain
DEPLOYMENTS_DIR=${MONOREPO_DIR}/packages/contracts-bedrock/deployments/getting-started
ROLLUP_CONFIG=${MONOREPO_DIR}/op-node/rollup.json
GENESIS_CONFIG=${MONOREPO_DIR}/op-node/genesis.json
L1_RPC_URL="http://awe.some.rpc"
SEQUENCER_RPC_URL="http://awe.some.seq.rpc"
L2_EXPLORER_URL="https://awesomescan.org"
```

```suggestion
MONOREPO_DIR=../optimism
SUPERCHAIN_TARGET=mainnet # L1 chain name
CHAIN_NAME=awesomechain # L2 chain name
DEPLOYMENTS_DIR=${MONOREPO_DIR}/packages/contracts-bedrock/deployments/getting-started
ROLLUP_CONFIG=${MONOREPO_DIR}/op-node/rollup.json
GENESIS_CONFIG=${MONOREPO_DIR}/op-node/genesis.json
PUBLIC_RPC="http://awe.some.rpc" # L1 RPC URL
SEQUENCER_RPC="http://awe.some.seq.rpc"
EXPLORER="https://awesomescan.org" # L1 block explorer URL
```

---

I went with option 2 for now, but followed up with [fix comments](https://github.com/ethereum-optimism/superchain-registry/pull/118/commits/8c02334d2706bc410123ca593bb8f7ec341d04a3). Let's make sure we are aligned on the meaning of these variables.